package.json

```diff
@@ -43,7 +42,7 @@
     "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-mocha": "^0.14.4",
+    "ember-cli-mocha": "emberjs/ember-mocha#master",
```

Comment:
interesting that this works... `ember-cli-mocha` vs `ember-mocha` 😉

Reply:
Uh, good catch! 😬

Reply:
Btw, do I still need `ember-cli-mocha` actually? Seems it only contains the blueprints nowadays, right?

Reply:
https://github.com/ember-cli/ember-cli-eslint/pull/225 and code like that is the problem and the reason why we're keeping the "wrapper" for now